src/auto/icon_info.rs

```diff
@@ -2,7 +2,7 @@
 // from gir-files (https://github.com/gtk-rs/gir-files)
 // DO NOT EDIT
 
-#[cfg(feature = "futures")]
+#[cfg(any(feature = "futures", feature = "dox"))]
```

Comment:
When "dox" is set but "futures" is unset, it won't compile because it lacks the future crates.

Reply:
Then futures crate should be included when dox is enabled. :)

Reply:
Ah that's what I was missing :)